lib/pdfkit/pdfkit.rb

```diff
@@ -129,7 +130,8 @@ def normalize_options(options)
       next if !value
 
       # The actual option for wkhtmltopdf
-      normalized_key = "--#{normalize_arg key}"
+      normalized_key = "#{normalize_arg key}"
```

Comment:
This works great, but just for clarity's/cleanliness' sake would you mind removing the string interpolation of the string?

```
normalized_key = normalize_arg key
```